UI/data/locale/en-US.ini

```diff
@@ -462,6 +462,22 @@ MissingFiles.AutoSearchText="OBS has found additional matches for missing files
 MissingFiles.NoMissing.Title="Missing Files Check"
 MissingFiles.NoMissing.Text="No files appear to be missing."
 
+# macOS permissions dialog
+MacPermissions.Title="Review App Permissions"
+MacPermissions.Description="OBS Studio requires your permission to be able to provide certain features. It is recommended to enable these permissions, but they are not required to use the app. You can always enable them later via the OBS Studio menu."
+MacPermissions.AccessGranted="Access Granted"
+MacPermissions.RequestAccess="Request Access"
+MacPermissions.OpenPreferences="Open %1 Preferences"
+MacPermissions.Item.ScreenRecording="Screen Recording"
+MacPermissions.Item.ScreenRecording.Details="OBS requires this permission to be able to capture your screen."
+MacPermissions.Item.Camera="Camera"
+MacPermissions.Item.Camera.Details="OBS requires this permission to be able to capture content from a webcam or capture card."
+MacPermissions.Item.Microphone="Microphone"
+MacPermissions.Item.Microphone.Details="OBS requires this permission to be able to capture your microphone."
+MacPermissions.Item.Accessibility="Accessibility"
+MacPermissions.Item.Accessibility.Details="OBS requires this permission so that you can use keyboard shortcuts (hotkeys) while other apps are focused."
+MacPermissions.Continue="Continue"
```

Comment:
It may also be worth having a message at the bottom instructing the user on how they can manually reopen this dialog later.

Reply:
I think moving that specific bit to the bottom would be recommended.

Reply:
Done
<img width="400" alt="image" src="https://user-images.githubusercontent.com/59806498/181052214-e4c745ad-e671-47f4-a708-3ab0b984d188.png">